includes/admin.form.inc

```diff
@@ -143,8 +143,18 @@ function islandora_entities_settings($form, &$form_state) {
       'islandora_entities_query_sort',
       ''
     ),
-    '#description' => t('Used to sort citations and theses on scholar profile pages. Should contain the solr field followed by a space and either "asc" or "desc". For example, "mods_originInfo_keyDate_yes_dateIssued_dt asc" will sort on the dateIssued field in ascending order.'),
+    '#description' => t('Used to sort citations and theses on scholar profile pages.'),
     '#required' => FALSE,
+    '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+  );
+
+  $form['islandora_entities_query_sort_direction'] = array(
+    '#type' => 'radios',
+    '#title' => t('Sort direction'),
+    '#description' => t('Sort direction for citations and theses on scholar profile pages.'),
+    '#default_value' => variable_get('islandora_entities_query_sort_direction', 0),
+    '#required' => FALSE,
+    '#options' => array(t('Ascending'), t('Descending')),
```

Comment:
Just as a note, the translation of those `0`/`1` indices to `asc` and `desc` could be avoided by making this an associative array with the keys being the sort directions, e.g.,

```php
// ...
'#options' => array(
  'asc' => t('Ascending'),
  'desc' => t('Descending'),
),
// ...
```

Otherwise it would probably have to be made more clear what `0` and `1` translate to through the use of something like module-level `const` statements